### PR TITLE
[backend] use cache storage service for workspace schema cache

### PR DESCRIPTION
**Pull request by Weiko**
Following this https://github.com/twentyhq/twenty/pull/4320
We are now removing MemoryStorage module in place of the new CacheStorageModule.
The only use case was to cache the workspace schema so I'm introducing a new WorkspaceSchema namespace

Screenshot 2024-03-06 at 16 05 26

### CODE REVIEW

File: packages/twenty-server/src/workspace/messaging/commands/crons/fetch-all-workspaces-messages.cron.pattern.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const fetchAllWorkspacesMessagesCronPattern = '*/10 * * * *';
+export const fetchAllWorkspacesMessagesCronPattern = '*/1 * * * *';
```
Comment (magrinj) [inline packages/twenty-server/src/workspace/messaging/commands/crons/fetch-all-workspaces-messages.cron.pattern.ts:1]: @Weiko Is it intended ?

**Review by magrinj — COMMENTED** [review]
LGTM !
Small point, maybe we should encapsulate the get, set of the cache into a service to properly type it and avoid key prefix error ?

File: packages/twenty-server/src/workspace/messaging/commands/crons/fetch-all-workspaces-messages.cron.pattern.ts (hunk 1 above)
Comment (Weiko) [reply packages/twenty-server/src/workspace/messaging/commands/crons/fetch-all-workspaces-messages.cron.pattern.ts:1]: @magrinj Oops no! I wanted to do some tests locally, not intended

**Comment by magrinj** [comment]
Not blocking for this PR, but yes we can add another layer later. From my understanding, namespace is type safe, but not what you provide in the key am I wrong ?

**Review by magrinj — APPROVED** [review]

**Pull request merged**